app/survey/survey.py

```diff
@@ -359,7 +451,7 @@ def generate_water_questions(suffixes, survey_questions):
                  " which meets drinking water standards, service water does not need to meet the same quality criteria. "
                  "Its primary function is to support specific activities without being directly consumed by humans.",
         "question_id": "2",
-        "possible_answers": ["yes", "no"],
+        "possible_answers": ["Yes", "No"],
```

Comment:
If you change "yes" to "Yes" here and not in the subquestion mapping below then the link between question's answer and subquestion is broken :) @JulianBarinton